### ISSUE TITLE
Sidebar

### ISSUE THREAD

**Issue by halimahLDN** [Greater-London-Authority/ldn-viz-tools#755]
**Sidebar**
- [x] Unstyled content in dark theme doesn't have enough colour contrast:
   `bg-color-palette-grey-100`
   ![image](https://github.com/user-attachments/assets/574a9995-8de4-45b8-a348-8ad029e5d2f0)

**SidebarSection**

- [x] Headling levels should only increase by one.
   - Main sidebar title is `h1`
   - Section Title is also `h1` (but think it should be `h2`)
   - Group title is `h3`
![image](https://github.com/user-attachments/assets/6977c162-b874-4496-aa64-9727f8aa9b43)

**SidebarTabs**
- [x] Tabs (highlighted) don't meet minimum colour contrast ratio threshold:
![image](https://github.com/user-attachments/assets/e9f2aa64-91dc-4403-87f5-fabf23af1116)

**SidebarFooter**
- [x] List should only directly contain `li`, `script`, or `template` elements:
![image](https://github.com/user-attachments/assets/29c8eb0c-5578-4433-a7cf-a4cb81d2b3c8)

**Screen reader/keyboard nav notes**
- [ ] Because of the order of DOM elements, 'More info' is read after the title and before the subtitle, which doesn't seem helpful
- [ ] Add [`role="region"`](https://www.w3.org/WAI/ARIA/apg/patterns/landmarks/examples/region.html) and `aria-label="Sidebar for [whatever it does]"` for better accessibility
   - I think this is the best option as, unlike other sidebars, the ones in our projects are imperative to the functionality of the site, so shouldn't be designated as an 'aside' (which as the name implies, [should be able to stand alone](https://www.w3.org/WAI/ARIA/apg/patterns/landmarks/examples/complementary.html))
- [ ] Sidebar announced after main content
   - Should the Sidebar be read before or after the main content?
- [x] [Tab options should be navigable](https://www.w3.org/WAI/ARIA/apg/patterns/tabs/#examples) with arrow keys (currently the tab key is used which is confusing because it makes it harder to tab into the focusable elements within the selected tab)
   - [x] Tabs should have a `tabindex="-1"`
- [x] Tab contents [should have a tabindex](https://www.w3.org/WAI/ARIA/apg/patterns/tabs/examples/tabs-automatic/#accessibilityfeatures) (`tabpanel`) where the first element is not focusable:
   ![Image](https://github.com/user-attachments/assets/051a7a90-1ee1-44c9-873f-138bd5c86812)
- [x] Tabs should have `aria-selected="true"` or `aria-selected="false"` attributes

**Comment by ChrisKnightLDN** [comment]
"Unstyled content in dark theme doesn't have enough colour contrast": This is as intended - text reads "				This is totally unstyled content and so needs to have extra classes to ensure correct padding etc. Use with extreme caution!"  maybe this text should be updated to read: "This is totally unstyled content and so needs to have extra classes to ensure correct **theming**, padding etc. Use with extreme caution!

**Comment by ChrisKnightLDN** [comment]
Typical Story at error for list
